### PR TITLE
Group Manager admin page

### PR DESCRIPTION
**Pull request by danieljames-dj**
This page is same as the Group Manager page in Leaders panel, but this will be in Board panel with all the teams, committees and councils.

### CODE REVIEW

File: app/controllers/api/v0/user_groups_controller.rb [hunk 1]
```diff
@@ -46,6 +46,19 @@ def index
       end
     end
 
+    # Temporary hack to support old system teams/committees.
+    if group_type == "teams_committees"
```
Comment (gregorbg) [inline app/controllers/api/v0/user_groups_controller.rb:50]: Use the constant from `UserGroup`?

File: app/webpacker/components/Panel/Leader/GroupsManager.jsx [hunk 2]
```diff
@@ -141,36 +141,36 @@ function GroupTable({ group }) {
...
+export default function GroupsManager({ loggedInUserId }) {
+  const { data: roles, loading, error } = useLoadedData(apiV0Urls.userRoles.listOfUser(
+    loggedInUserId,
+    'groupName', // Sort params
+    { isActive: true, isGroupHidden: false, status: 'leader' },
+  ));
+
+  if (!loading && roles.length === 0) return <p>You are not a leader of any groups.</p>;
+  if (loading) return <Loading />;
+  if (error) return <Errored />;
```
Comment (gregorbg) [inline app/webpacker/components/Panel/Leader/GroupsManager.jsx:174]: You should do the `loading` and `error` checks first, and then below these two simply do `if (roles.length === 0)`

**Review by gregorbg — CHANGES_REQUESTED** [review]
Only small nitpicks :D

File: app/controllers/api/v0/user_groups_controller.rb (hunk 1 above)
Comment (danieljames-dj) [reply app/controllers/api/v0/user_groups_controller.rb]: Done

File: app/webpacker/components/Panel/Leader/GroupsManager.jsx (hunk 2 above)
Comment (danieljames-dj) [reply app/webpacker/components/Panel/Leader/GroupsManager.jsx]: Oops ya, done

**Review by gregorbg — APPROVED** [review]

**Pull request merged**